.swift-version

```diff
@@ -1 +1 @@
-5.0
+4.2
```

Comment:
From the OAuthSwift's maintainers POV, I think it would be easier to get this PR merged if we targeted their newest release (`master` branch). I'd assume they keep older versions in maintenance mode.

And I think that's fine for us because I imagine we'll migrate to Swift 5 just before or after releasing 1.8.